### PR TITLE
Remove sent_transactions table.

### PR DESCRIPTION
**Pull request by erikjohnston**
It does nothing.

### CODE REVIEW

**Comment by NegativeMjark** [comment]
Sure. LGTM

File: synapse/storage/transactions.py [hunk 1]
```diff
@@ -338,59 +234,3 @@ def _get_destinations_needing_retry(self, txn):
...
-        except:
-            logger.exception("Failed to persist transactions!")
-
-    def _cleanup_transactions(self):
-        now = self._clock.time_msec()
-        month_ago = now - 30 * 24 * 60 * 60 * 1000
-        six_hours_ago = now - 6 * 60 * 60 * 1000
-
-        def _cleanup_transactions_txn(txn):
-            txn.execute("DELETE FROM received_transactions WHERE ts < ?", (month_ago,))
```
Comment (NegativeMjark) [inline synapse/storage/transactions.py]: Are you sure about the ``received_transactions``?

**Comment by NegativeMjark** [comment]
I think received_transactions is a write-only table as well. Should we remove it while we are at it?

**Comment by erikjohnston** [comment]
> I think received_transactions is a write-only table as well. Should we stop writing to it as well while we are at it?

It's used to short circuit duplicate transactions

**Pull request merged**